src/gc/gcinterface.h

```diff
@@ -546,16 +546,16 @@ class IGCHeap {
     virtual HRESULT Initialize() = 0;
 
     // Returns whether nor this GC was promoted by the last GC.
-    virtual BOOL IsPromoted(Object* object) = 0;
+    virtual bool IsPromoted(Object* object) = 0;
 
     // Returns true if this pointer points into a GC heap, false otherwise.
-    virtual BOOL IsHeapPointer(void* object, BOOL small_heap_only = FALSE) = 0;
+    virtual bool IsHeapPointer(void* object, bool small_heap_only = false) = 0;
```

Comment:
>IsHeapPointer [](start = 17, length = 13)

what about callers of these methods? eg, we call IsHeapPointer with TRUE in object.cpp